utils/scripts/testconfig/test-variables.yml

```diff
@@ -52,3 +52,7 @@ GITHUB_USER:
   EMAIL: GITHUB_EMAIL
   USERNAME: GITHUB_USERNAME
   TOKEN: GITHUB_TOKEN
+STARBURST:
```

Comment:
Missing 
  OBS_CLIENT_SECRET:  
  OBS_CLIENT_ID
  OBS_URL:  
  OBS_TOKEN_URL:

Reply:
these are in the other PR: https://github.com/red-hat-data-services/ods-ci/pull/635/files#diff-c5322351257b41d6672b2a6ca4dfc126281fe481369b2db4a170e223e6c8c00d